Resources/public/templates/studiopresentation.hbt

```diff
@@ -1,22 +1 @@
-<p>You don't have eZ Studio Installed :(</p>
-
-<h1>Learn more about eZ Studio</h1>
-
-<p>
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin facilisis tristique felis, in euismod
-    ipsum sollicitudin vitae. Morbi pellentesque ut purus non placerat. Vivamus sollicitudin condimentum
-    nunc, eget euismod dolor. Etiam ultricies sit amet dui vel sagittis. Maecenas vitae fermentum urna.
-    Sed aliquam lacinia ante, ac convallis justo pulvinar ac. Nunc pellentesque lobortis diam, nec
-    placerat sapien pulvinar ac. Nullam euismod mattis dictum. Suspendisse consectetur, nisl vel
-    porttitor bibendum, eros nisi placerat est, ut mattis magna lorem ac dolor. Aliquam viverra sit
-    amet purus eu tincidunt.
-</p>
-
-<p>
-Quisque ut tincidunt elit. Pellentesque nulla felis, vestibulum non eros vitae, lacinia luctus odio.
-    Etiam dapibus metus eget bibendum mattis. Donec lectus risus, hendrerit non condimentum et,
-    venenatis nec neque. Etiam sed varius arcu. Nulla ullamcorper luctus augue, blandit tempor neque
-    lobortis sit amet. Mauris ac eleifend augue. Etiam condimentum auctor ex, nec vehicula nunc aliquam
-    et. Quisque sagittis sed mi maximus rutrum. Proin eu enim non neque viverra iaculis. In hac habitasse
-    platea dictumst. Aenean euismod nunc urna, id iaculis nisl gravida vel.
-</p>
+<iframe class="studio-presentation-iframe" src="about:blank" width="100%" height="100%"></iframe>
```

Comment:
please always prefix the class names with `ez-`. Usually, we also avoid the tag name in the class name. And setting the size of the iframe like that is very HTML4ish, please use the CSS for that.